app/Models/BookingRequest.php

```diff
@@ -33,6 +33,17 @@ class BookingRequest extends Model
         'onsite_contact' => 'array',
     ];
 
+    /**
```

Comment:
Convention: Constants should always be defined before any non-constants

Reply:
updated